packages/skiyee-uni-ui/src/constants/accordion.ts

```
@@ -0,0 +1,34 @@
+/**
+ * Accordion 手风琴常量
+ *
+ * @author sKy(skiyee)
+ * @qq 319619193
+ * @github https://github.com/skiyee
+ */
+
+import type { InjectionKey } from 'vue'
+
+import type { SkAccordionItemUcvProps, SkAccordionUcvProps } from '../styles'
```

ESLint

[error] 'SkAccordionItemUcvProps' is defined but never used.

(unused-imports/no-unused-imports)